src/templates/users/_edit.html

```diff
@@ -157,25 +157,38 @@
 				</div>
 
 				{% if not isNewAccount or CraftEdition == CraftPro %}
-					<div id="profile" class="hidden">
-						{% if not isNewAccount %}
-							{{ forms.field({
-								label: "Photo"|t
-							}, photoInput) }}
-
-							<hr>
-						{% endif %}
+					{% set customLayout = craft.fields.getLayoutByType('User') %}
+					{% for tab in customLayout.tabs %}
+						<div id="custom-{{ tab.id }}" class="hidden">
+
+							{# show profile image on first tab #}
+							{% if not isNewAccount and loop.first %}
+								{{ forms.field({ label: "Photo"|t }, photoInput) }}
+								<hr>
+							{% endif %}
```

Comment:
By putting the Photo field here, it’s only going to show up if the user field layout actually has any tabs (it might not). In Craft 3 we moved the Photo field [over to the Account tab](https://github.com/craftcms/cms/blob/f2617ae5a6218329ceb42db2586500814437eafc/src/templates/users/_edit.html#L60-L64), after the First/Last Name fields. Can you move it there for this PR as well? (The following Photo field starting on line 177 can be deleted as well, in that case.)

Reply:
No. Look at the `{% else %}` part on line 174 below. That's the fallback part where the photo url is still there if you have no tabs configured. When you do, it's added on the first tab you added.

Reply:
Photo url either is part of the account. It's a additional thing to the user (in my opinion)

Reply:
Ah I get it, I assumed the `{% else %}` was for the `{% if %}` on line 159, not the `{% for %}`.

Still, I would prefer to have this be consistent with Craft 3, rather than create a Profile tab if there are no custom field layout tabs.

Reply:
I do understand that. But maybe you should reconsider placing first and last name fields too. Since they are not necessary for the account to login. The account-details are username/email and password and the others are profile-data :-) but if you really really really want that, I will change it.

Reply:
Eventually we will be making the placement of _all_ user fields (including Username, First/Last Name, Photo, custom fields, etc.) customizable from the Field Layout Manager, and at that point it will be up to the admin to decide how they want to organize everything. Until then, I’m OK with it the "Account" tab label being slightly un-semantic.

Reply:
Okay. I will move the photo field then and remove the support for the profile tab when you not have any tabs configured. Will update it a.s.a.p.
Btw. permissions for tabs and fields would be cool too then :) and different setup for different types of users and usergroups.

Reply:
@brandonkelly done! :-)

Reply:
And you tested right?

Reply:
Of course I did. Check screenshots:

![image](https://user-images.githubusercontent.com/398492/34894863-590dc5e6-f7e3-11e7-9264-bd91be54427e.png)

![image](https://user-images.githubusercontent.com/398492/34894868-5feaa5dc-f7e3-11e7-88e5-5fd35fb90dcb.png)